### DISCUSSION TITLE
My VDH license appears to be locked and I do not get any emails. So how do I get it unlocked

### DISCUSSION THREAD

**Discussion by VinceB662** [aclap-dev/video-downloadhelper#1133] [Help and Q&A]
### Your question

How do I get my valid VDH license unlocked? 

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"locked":true,"key":"e22fb2b39f7a40c5"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 3
}

**Comment by jcfrog** [comment]
Hi,
You should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Comment by VinceB662** [comment]
Hi,Did not receive an email to reset my counter. However my license now is validated Thank you for that!By the way, the original problem of VDH not detecting videos is still there except when I disable my Norton VPN.Is there still an issue here? 

    On Sunday, August 11, 2024 at 04:31:10 PM EDT, jcfrog ***@***.***> wrote:   

 


Hi,
You should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Reply by GrampaWildWilly** [reply]
Sounds to me (a NON-employee of VDH, just another user) like the E-mail registered for you in the VDH data base is not the one you are using now.  You should send @jcfrog Jérôme an E-mail so he can verify whether the address you are using matches what he's got.
